Dockerfile

```diff
@@ -28,8 +28,8 @@ COPY --from=build /app/.nuxt ./.nuxt
 COPY --from=build /app/static ./static
 
 # Install runtime dependencies
-RUN yarn install --production --no-lockfile
+RUN npm install --production --no-save --no-audit
```

Comment:
Why not go `npm ci` in that case? :)

Reply:
``npm ci`` fetches packages from the lockfile (``pacakge-lock.json``). Here, we're installing the runtime dependencies for SSR, so there's no lockfile, we're doing a fresh installation from ``package.json``